Makefile.common_go

```diff
@@ -50,13 +50,13 @@ spell:
 # NB: "third_party" only exists for automate-gateway, but no harm having it for other dirs here.
 semgrep:
 # uncomment if custom rules beyond automate-ui ever get added
-#	semgrep --config $(REPOROOT)/semgrep --exclude third_party
-	semgrep --config https://semgrep.dev/p/r2c-ci --exclude third_party --autofix
+#	semgrep --config $(REPOROOT)/semgrep --exclude third_party --exclude *_test.go --exclude *.pb.go
+	semgrep --config https://semgrep.dev/p/r2c-ci --exclude third_party --exclude *_test.go --exclude *.pb.go
```

Comment:
Am I wrong with the idea that semgrep has a config file, too? Using that instead, we may be able to share it between the Makefile and the BK pipeline definition...?

Or, rather, and perhaps preferrably, we could use the make target from within the BK job, I would think!

Reply:
Ignore me, this is pretty much what we're doing, I believe. It took me a while to digest what's going on here. 😄